src/accelerate/checkpointing.py

```diff
@@ -214,3 +230,322 @@ def load_custom_state(obj, path, index: int = 0):
     load_location = f"{path}/custom_checkpoint_{index}.pkl"
     logger.info(f"Loading the state of {get_pretty_name(obj)} from {load_location}")
     obj.load_state_dict(torch.load(load_location, map_location="cpu"))
+
+
+def dtype_byte_size(dtype):
```

Comment:
I think this already exists in utils/modeling ;-)

Reply:
my bad ;-) [8d853b4](https://github.com/huggingface/accelerate/pull/1641/commits/8d853b4c7918360b91877737c363b5722b4c8e48)